plugins/plugin-shiki/src/node/highlight.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  transformerCompactLineOptions,
+  transformerNotationDiff,
+  transformerNotationErrorLevel,
+  transformerNotationFocus,
+  transformerNotationHighlight,
+} from '@shikijs/transformers'
+import { customAlphabet } from 'nanoid'
+import type { ShikiTransformer } from 'shiki'
+import {
+  bundledLanguages,
+  getHighlighter,
+  isPlainLang,
+  isSpecialLang,
+} from 'shiki'
+import { colors as c, logger } from 'vuepress/utils'
```

Comment:
why as c here?

Reply:
oh, I am used to abbreviating picocolors as `c`, I will change it back to `colors`.